src/extensions/vb/Microsoft.DotNet.UpgradeAssistant.Steps.VisualBasic/VisualBasicProjectUpdaterStep.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Steps.VisualBasic
+{
+    /// <summary>
+    /// Upgrade step that applies changes specific to helping customers
+    /// working on VB projects. Includes things that help to fix the "My."
+    /// namespace and other things specific to VB.
+    /// </summary>
+    public class VisualBasicProjectUpdaterStep : UpgradeStep
+    {
+        public override string Title => "Update Visual Basic project";
+
+        public override string Description => "Update Visual Basic projects using registered VB updaters";
+
+        public override string Id => WellKnownStepIds.VisualBasicProjectUpdaterStepId;
+
+        public override IEnumerable<string> DependsOn { get; } = new[]
+        {
+            // Project should be backed up before changing source
+            WellKnownStepIds.BackupStepId,
+
+            // Template files should be added prior to changing source (since some code fixers will change added templates)
+            WellKnownStepIds.TemplateInserterStepId,
+
+            // Project should have correct TFM
+            WellKnownStepIds.SetTFMStepId,
+        };
+
+        public override IEnumerable<string> DependencyOf { get; } = new[]
+        {
+            WellKnownStepIds.NextProjectStepId,
+        };
+
+        public VisualBasicProjectUpdaterStep(ILogger<VisualBasicProjectUpdaterStep> logger)
+            : base(logger)
+        {
+            if (logger is null)
+            {
+                throw new ArgumentNullException(nameof(logger));
+            }
+
+            // Add a sub-step for each VB specific fix
+            SubSteps = new List<UpgradeStep>() { new EnableMyDotSupportSubStep(this, logger) };
```

Comment:
If this only has one sub-step, why not just make the sub-step the parent step and get rid of the nesting?

Reply:
I'm under the impression this will grow as we begin to resolve other issues open for the VB area. I can setup as a single step and we can grow as needed but was trying to lay the ground work for future PRs.

Will leave as-is. Please unresolve if you think we should defer this work until there are actually 2 steps.